packages/tpetra/core/src/Tpetra_Util.hpp

```diff
@@ -629,7 +629,7 @@ namespace Tpetra {
   ///         val.begin (), val.end ());
   /// \endcode
   template<class IT1, class IT2>
-  KOKKOS_INLINE_FUNCTION void
+  void
```

Comment:
Huh, I remember this causing warnings, but had thought that we were actually calling `merge2` in a GPU kernel somewhere.  Guess not! : - )

Reply:
I was also sure there was a good reason for it, but I checked every call site.